apps/etterlatte-behandling/src/test/kotlin/behandling/BehandlingServiceImplTest.kt

```diff
@@ -264,7 +268,7 @@ class BehandlingServiceImplTest {
             }
         val behandlingHendelserKafkaProducer =
             mockk<BehandlingHendelserKafkaProducer> {
-                every { sendMeldingForHendelse(any(), any()) } returns Unit
+                every { this@mockk.sendMeldingForHendelseMedDetaljertBehandling(any(), any()) } returns Unit
```

Comment:
Er ikke `this@mockk` implisitt her om du fjerner det?

Reply:
Jo er noe autocomplete som har sneket seg inn 🤔

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2255/commits/120f0437d90c2b9a53fa737b19dd1b2483b9abca